### ISSUE TITLE
failed to dial to edge with quic

### ISSUE THREAD

**Issue by astanoszek** [DarkAssassinUA/cloudflared#1]
When I try to establish the tunnel using your binary on my router quite old I get the following error:

````
2024-01-18T20:22:19Z ERR Failed to create new quic connection error="failed to dial to edge with quic: INTERNAL_ERROR: read udp [::]:38343: recvmmsg: function not implemented" connIndex=0 ip=198.41.192.107
````

Maybe you have the idea what should I do to fix the problem?

**Comment by Le-Maxime** [comment]
> When I try to establish the tunnel using your binary on my router quite old I get the following error:
> 
> ```
> 2024-01-18T20:22:19Z ERR Failed to create new quic connection error="failed to dial to edge with quic: INTERNAL_ERROR: read udp [::]:38343: recvmmsg: function not implemented" connIndex=0 ip=198.41.192.107
> ```
> 
> Maybe you have the idea what should I do to fix the problem?

Hi!
Check this - https://github.com/cloudflare/cloudflared/issues/1085

**Comment by astanoszek** [comment]
Thank you. Meanwhile I noticed that http2 protocol works, so I bypassed the problem.

Thank you for publishing your binary!
